### DISCUSSION TITLE
PP-OCRv4 在线体验可以识别，但是自己用PP-OCRv4模型无法识别

### DISCUSSION THREAD

**Discussion by ycw01** [PaddlePaddle/PaddleOCR#13571] [Q&A]
![data_bb_val_lock_1](https://github.com/user-attachments/assets/67f044dd-8d90-4dd3-83a5-b89efaeac774)
![微信截图_20240802094829](https://github.com/user-attachments/assets/907057e8-294a-46b2-ab61-66ed3184cb2e)

from paddleocr import PaddleOCR

ocr = PaddleOCR(use_angle_cls=False, use_gpu=False, lang="ch",
                det_model_dir="resources/ch_PP-OCRv4_det_server_infer")
r1 = ocr.ocr("data_bb_val_lock_1.jpg", cls=True)
print(r1)
![1](https://github.com/user-attachments/assets/0f3d1f69-6cab-4410-b1bd-79d4075ff694)

**Comment by SWHL** [comment]
这个我们要确定一下在线的demo是部署的哪个？因为不是PMC成员部署的
